src/components/Footer/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import styles from "./Footer.module.scss"
```

Comment:
no need to have named import here. You can just import
import './footer.module.scss'

Reply:
imported "./footer.module.scss" directly into Footer component